### PR TITLE
add more feelimit to tron trc20

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Add trc20 feelimit to allow user to send trc20 with trx and not energy only

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

Screenshot 2022-12-06 at 15 44 27

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/api/Tron.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ export const createTronTransaction = async (
   if (tokenType === "trc20" && tokenId) {
     const txData: SmartContractTransactionData = {
       function_selector: "transfer(address,uint256)",
-      fee_limit: 10000000,
+      fee_limit: 20000000,
```
Comment (lvndry) [inline libs/ledger-live-common/src/api/Tron.ts:122]: why this value ?

File: libs/ledger-live-common/src/families/tron/bridge/js.ts [hunk 2]
```diff
@@ -764,8 +764,7 @@ const getTransactionStatus = async (
     if (
       account.type === "TokenAccount" &&
       account.token.tokenType === "trc20" &&
-      energy.eq(0) &&
-      a.spendableBalance.lt(1000000)
+      energy.lt(29650) // temporary value corresponding to usdt trc20 energy
```
Comment (lvndry) [inline libs/ledger-live-common/src/families/tron/bridge/js.ts:767]: Why this value ?

Comment (henrily-ledger) [reply libs/ledger-live-common/src/families/tron/bridge/js.ts:767]: It's about this price of energy if you are doing a TRC20 USDT transfer ! But yeah it's kind of weird we do not have a simple way to fetch this data. ATM I just take this data from explorer

File: libs/ledger-live-common/src/api/Tron.ts (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledger-live-common/src/api/Tron.ts:122]: It was discuss by Fabrice https://ledger.slack.com/archives/CRD18A0H5/p1670237662561249?thread_ts=1670221369.653679&cid=CRD18A0H5

We want to upgrade the fee_limit allowing user to be able to do a transaction even tho they do not have enough Energy.

**Review by haammar-ledger — APPROVED** [review]

**Comment by haammar-ledger** [comment]
Superseded by https://github.com/LedgerHQ/ledger-live/pull/2104

**Pull request closed without merging**